example/pubspec.yaml

```diff
@@ -31,11 +32,11 @@ dependencies:
     sdk: flutter
 
   flutter_audio_waveforms:
-    path: /Users/rutviktak/flutter_projects/flutter_plugins/flutter_audio_waveforms/ # 1.0.0+1
+    path: /home/wolverine/projects/flutter-projects/flutter packages/flutter_audio_waveforms
```

Comment:
This is pointing to your machine path, try reverting this to the old one.

Reply:
fixed